variables/_svgs.pcss

```diff
@@ -23,6 +23,17 @@
 	}
 }
 
+/* -----------------------------------------------------------------------------
+ * SVG - Arrow Left
+ * ----------------------------------------------------------------------------- */
+
+@svg-load arrow-left url('../icons/arrow.svg') {
```

Comment:
is this arrow the same as arrow right but pointing left? if so, can we name this file so it matches the same format (`arrow-left.svg`)?